docs/download.md

```diff
@@ -100,6 +100,10 @@ The list of all supported operations is provided [here](supported_ops.md).
 For a detailed list of changes, please refer to the
 [CHANGELOG](https://github.com/NVIDIA/spark-rapids/blob/main/CHANGELOG.md). 
 
+**_Note:_** Using the CUDA 450.80.02 driver in combination with the CUDA 10.1 or 10.2 toolkit may
+result in long read times when reading a file that is snappy compressed.  If using the CUDA
+450.80.02 driver we recommend running with the CUDA 11.0 toolkit.  This issue is resolved in the 0.5
```

Comment:
do we want to change 0.5 to v0.5.0 to match syntax above?

Reply:
Yes, updated to 0.5.0